### ISSUE TITLE
Some ParticlesMaterial property descriptions were lost

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#57885]
### Godot version

8161e28

### System information

W10

### Issue description

I noticed that after #50434 some ParticlesMaterial that had descriptions don't have them anymore or they lost some details. E.g. `angle` had a detailed description how it works, now the new min/max angle property just say that it's angle. `damping` had some brief description, but it doesn't anymore.

I think it's worth going over the changes in #50434  and copying the descriptions to the respective new properties, as they are still relevant.

### Steps to reproduce

1. Go to ParticlesMaterial documentation (CPUParticles have the same problem probably)
2. See many missing descriptions that were there before

**Comment by Exentric24** [comment]
Can you assign me this issue @KoBeWi? I would love to work on this.
And I have just 2 questions;
1) Do I only have to locate where all the descriptions are missing or should I also add the descriptions?
2) And if I should add the descriptions, where can I find them or something related to them?

**Comment by akien-mga** [comment]
> > Can you assign me this issue @KoBeWi? I would love to work on this.
> 
> For future reference, we only use the assignees feature for core contributors (and we don't always use it in the first place). Since contributors can leave at any time for any reason, we'd rather avoid a situation where nobody is working on an issue because it's assigned to someone who is no longer active.

And more importantly: users who are not part of the @godotengine organization *cannot* be assigned to issues on GitHub.

**Comment by golddotasksquestions** [comment]
@Exentric24 should still be able to make the necessary PR though, even without being assigned to it.
